### PR TITLE
Snlp_4

### LINKED ISSUE #4: change "Sample Data" folder name to "samples"

### PR DESCRIPTION
**Pull request by VarmaHub**
Issue 4: Renaming Sample Folder to samples.

Edit: resolves #4

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea/
```
Comment (BurhanOtour) [inline .gitignore]: This line is redundant. No .idea/ folder exists in the root directory.  @VarmaHub

Comment (VarmaHub) [reply .gitignore]: There are specific paths are mentioned for .idea. Instead, we could ignore all subfolders of it with this line.

Comment (BurhanOtour) [reply .gitignore]: @VarmaHub No we can't simply ignore all idea files such as compiler.xml

Comment (VarmaHub) [reply .gitignore]: Done!

Comment (VarmaHub) [reply .gitignore]: Oh! I see. I have reverted the commit ignoring this change.

**Pull request merged**